aldryn_faq/tests/test_plugins.py

```diff
@@ -86,7 +102,7 @@ def test_plugin(self):
         self.question1.save()
         request = self.get_page_request(
             page1, self.user, None, lang_code='en', edit=False)
-        context = RequestContext(request, {})
+        context = RequestContext(request, {'request': request})
```

Comment:
It's a bit strange that sometime you use a single- and sometimes a double-quote.